src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -26,6 +26,7 @@ import spray.routing.RequestContext
 import scala.collection.JavaConversions._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Try}
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Ah, yes, I remember why I didn't do that. The method that needs the implicit is `status: Future[SubsystemStatus]` which is abstract in `ReportsSubsystemStatus`. I did not want to change the signature  across all of the implementations so I chose this path. If there's a different way to do this, I'm all ears. Also, `HttpGoogleServicesDAO` is an object, not a class like the other DAO implementations we have and I can't add the implicit to the object declaration.